pallets/ddc-verification/src/lib.rs

```diff
@@ -1393,6 +1483,47 @@ pub mod pallet {
 			)))
 		}
 
+		pub(crate) fn fetch_sub_trees(
+			cluster_id: &ClusterId,
+			era_id: DdcEra,
+			customer_activities: Vec<CustomerActivity>,
+			min_nodes: u16,
```

Comment:
I think renaming `min_node` to something like `quorum` would make the code more clear. We have too many definitions of nodes :) This one stays for a number of DAC nodes as far as I understand